calico/_includes/charts/calico/templates/calico-node.yaml

```diff
@@ -552,10 +560,17 @@ spec:
           hostPath:
             path: /run/xtables.lock
             type: FileOrCreate
-        - name: sysfs
+        - name: sys-fs
           hostPath:
             path: /sys/fs/
             type: DirectoryOrCreate
+        - name: bpffs
+          hostPath:
+            path: /sys/fs/bpf
+            type: Directory
+        - name: init-proc
```

Comment:
This is a very odd-looking mount, let's add a comment explaining why we're doing that.